adapters/adtrgtme/adtrgtme.go

```diff
@@ -32,9 +32,24 @@ func (v *adapter) MakeRequests(
 	[]*adapters.RequestData,
 	[]error,
 ) {
+	var ext openrtb_ext.ExtImpAdtrgtme
 	var requests []*adapters.RequestData
 	var errors []error
 
+	for i, imp := range openRTBRequest.Imp {
```

Comment:
When ranging over a slice, two values are returned for each iteration. The first is the index, and the second is a copy of the element at that index. 
I think you intend to pass the openRTBRequest.Imp directly in the for loop and not the copy of the `openRTBRequest.Imp[i]`. So, I would suggest to access imp using index instead. Here is the sample - 
```
for i, _ := range openRTBRequest.Imp {
\\ access imp using openRTBRequest.Imp[i] <-- this would be the actual openRTBRequest.Imp[i] and not copy of openRTBRequest.Imp[i]

}

Reply:
It doesn't matter to us. We will receive impressions with the same extensions and need to get any of them. And remove extensions from the outgoing HTTP requests to our server.

Reply:
For this reason, we overwrite it in the same variable `ext`
```
var ext openrtb_ext.ExtImpAdtrgtme
....
for i, imp := range openRTBRequest.Imp {
var err error
ext, err = getImpressionExt(&imp)
```